docs/docusaurus/docs/core/set_up_a_gx_environment/_install_gx/_gx_cloud_installation.md

```diff
@@ -0,0 +1,7 @@
+GX Cloud provides a web interface for using GX to validate your data without creating and running complex Python code.  However, GX 1.0 is also capable of connecting to a GX Cloud account should you wish to engage in further customization or automation of your workflows through Python scripts.
```

Comment:
```suggestion
GX Cloud provides a web interface for using GX to validate your data without creating and running complex Python code. However, GX 1.0 can connect to a GX Cloud account if you want to customize or automate your workflows through Python scripts.
```
Should you be using a variable for GX 1.0?

Find and remove all instances of _wish_.

![image](https://github.com/great-expectations/great_expectations/assets/104205257/8f5c993b-e84c-4b27-b9b2-6357510e4419)